server/config/config.go

```diff
@@ -197,14 +197,14 @@ func fixConfig(c *Config) *Config {
 // nameWithoutVersionScope e.g. faker
 func extractPackageName(packageName string) (fullNameWithoutVersion string, scope string, nameWithoutVersionScope string) {
 	paths := strings.Split(packageName, "/")
```

Comment:
i think we don't need to parse the `packageName` here, since the handler already parsed it:

https://github.com/esm-dev/esm.sh/blob/main/server/esm_handler.go#L486
<img width="851" alt="Screenshot 2024-02-01 at 18 43 28" src="https://github.com/esm-dev/esm.sh/assets/2883484/ceab814d-865c-4397-b98f-4796d95ceb29">
would you like to update it? thanks 🙏

Reply:
Yes. That could work. So I'll update the code to use that.

But wouldn't it make sense to also update the `extractPackagename`?

Reply:
> But wouldn't it make sense to also update the extractPackagename?

yes! please

Reply:
I needed the scope and package name, so it made sense to keep that part of the code. Don't you think?

Reply:
yeah, makes sense